### PR TITLE
Add secrets.json to gitignore.

### PR DESCRIPTION
**Pull request by milkypostman**
Includes a good sorting.

### CODE REVIEW

**Comment by jiayliu** [comment]
Isn't secrets.json in out/app_engine?

**Comment by milkypostman** [comment]
yes, but since out/app_engine gets wiped everytime you build, you have to setup some sort of loop which recopies it when you make a new change... I use the following command to manage this while I'm doing dev,

```
while true; do grunt build && (([ -e secrets.json ] && cp secrets.json out/app_engine) || true) && ~/src/google_appengine/dev_appserver.py out/app_engine; done
````

normally `grunt build` is enough cause dev_appserver.py will restart automatically but I've found times that this didn't work as nicely as I wanted.

Regardless, I could keep the secrets.json outside the repo, but I find it's nice to just have it there and never commit it, rather than having it sit somewhere in my home directory or something.

**Comment by milkypostman** [comment]
tl;dr: the cost of adding one line in `.gitignore` is worth it to make sure no one accidentally commits it regardless of their dev habits :smile:

**Comment by jiayliu** [comment]
I'm fine with it, but I also feel it's a little arbitrary: one can put
secrets.json in any subdir, or any other file in root.

On Fri, Feb 20, 2015 at 5:44 PM, Donald Ephraim Curtis <
notifications@github.com> wrote:

> tl;dr: the cost of adding one line in .gitignore is worth it to make sure
> no one accidentally commits it regardless of their dev habits [image:
> :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/90#issuecomment-75349264>.
>



-- 
Jiayang Liu

**Comment by milkypostman** [comment]
this will ignore it in any directory.

**Comment by jiayliu** [comment]
ok, lgtm

On Fri, Feb 20, 2015 at 6:00 PM, Donald Ephraim Curtis <
notifications@github.com> wrote:

> this will ignore it in any directory.
>
> On Fri Feb 20 2015 at 5:58:30 PM Jiayang Liu <notifications@github.com>
> wrote:
>
> > I'm fine with it, but I also feel it's a little arbitrary: one can put
> > secrets.json in any subdir, or any other file in root.
> >
> > On Fri, Feb 20, 2015 at 5:44 PM, Donald Ephraim Curtis <
> > notifications@github.com> wrote:
> >
> > > tl;dr: the cost of adding one line in .gitignore is worth it to make
> sure
> > > no one accidentally commits it regardless of their dev habits [image:
> > > :smile:]
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/webrtc/apprtc/pull/90#issuecomment-75349264>.
> > >
> >
> >
> >
> > --
> > Jiayang Liu
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/webrtc/apprtc/pull/90#issuecomment-75349996>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/90#issuecomment-75350115>.
>



-- 
Jiayang Liu
